<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-A38V3DO6" failures="1" tests="4" name="runner.TestRunner" time="21.270" errors="0" timestamp="2023-08-24T14:05:44 IST" skipped="0">
  <testcase classname="runner.TestRunner" name="runScenario" time="8.372">
    <failure message="Failed due to unccessful Logged in message " type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Failed due to unccessful Logged in message 
at org.testng.Assert.fail(Assert.java:110)
at stepDefinitions.login_stepDeff.user_is_navigated_to_home_page(login_stepDeff.java:80)
at âœ½.User is navigated to home page(file:///C:/Users/Sanjay%20Prasad/eclipse-workspace/allurecucumberProject2/src/test/resources/features/loginPage.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="4.139"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="3.739"/>
  <system-out/>
  <testcase classname="runner.TestRunner" name="runScenario" time="5.020"/>
  <system-out/>
</testsuite> <!-- runner.TestRunner -->
